proto/storage/virtual_machine.proto

```diff
@@ -0,0 +1,46 @@
+syntax = "proto3";
+
+package storage;
+
+import "google/protobuf/timestamp.proto";
+import "storage/image.proto";
+
+message VirtualMachine {
```

Comment:
Do we have a data model for this some place or are we just going on what images and nodes have?

Reply:
We should probably have a good definition of what data will be gathered and compiled.  Scan stuff is likely the same, but we have flexibility on how we model that.

Reply:
Would you mind if I added this as a TODO in the code here to follow up and ensure we've considered cleaning this up before we mark the API epic as complete?